src/lib/events.js

```diff
@@ -3,13 +3,13 @@
 export function emit(listeners, ...args) {
   let type = args.shift();
 
-  if (listeners[type]) {
-    listeners[type].slice().forEach(
+  if (listeners['*']) {
+    listeners['*'].slice().forEach(
       listener => listener.apply(this, args));
   }
 
-  if (listeners['*']) {
-    listeners['*'].slice().forEach(
+  if (listeners[type]) {
+    listeners[type].slice().forEach(
       listener => listener.apply(this, args));
   }
```

Comment:
I'm using the `*` event listener to add the name of the current app in `amendError`. I need it to run before any other event handlers, so that if any of them logs the error directly to stdout, it already has the amended message.

I'm sure there's a better way of achieving this, but this seemed easy and harmless enough :)  Any ideas?